.github/workflows/lang-npgsql.yml

```diff
@@ -85,7 +85,7 @@ jobs:
 
       # https://github.com/codecov/codecov-action
       - name: Upload coverage results to Codecov
-        uses: codecov/codecov-action@v4
+        uses: codecov/codecov-action@v5
```

Comment:
Not sure if this has any value here. Maybe let's check why Codecov has been enabled on a Npgsql project, and possibly remove it again.